src/haconiwa/task/submit.py

```
@@ -0,0 +1,628 @@
+            
+            logger.info(f"✅ Successfully submitted task: {title}")
+            logger.info(f"📁 Worktree created at: {worktree_path}")
+            logger.info(f"👤 Assigned to: {assignee}")
+            
+            return task
+            
+        except TaskSubmitError:
+            raise
+        except Exception as e:
+            logger.error(f"Unexpected error during task submission: {e}")
+            raise TaskSubmitError(f"Failed to submit task: {str(e)}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)